README.md

```diff
@@ -2,106 +2,36 @@
 
 # ApiCenter
 
-ApiCenter is a repository for your OpenAPI and GraphQL specifications. Through the web interface you can upload specifications, interact with them through the [SwaggerUI](https://swagger.io/tools/swagger-ui/) / [GraphiQL](https://github.com/graphql/graphiql), and search their contents.
+ApiCenter is a repository for OpenAPI and GraphQL specifications. Through the web interface you can upload & download specifications, interact with them through the [SwaggerUI](https://swagger.io/tools/swagger-ui/) / [GraphiQL](https://github.com/graphql/graphiql), and search their contents.
 
-## How to use
+More information and troubleshooting is on the [wiki](https://github.com/TNG/ApiCenter/wiki).
```

Comment:
Some screenshots, maybe?

Reply:
I decided against this, thinking that for the README to render locally, these images would have to be committed as part of the project. And the screenshots aren't really static assets, they'd need updating in response to cosmetic UI changes, which in my mind would just create noise in the commit history. It's much simpler to manage screenshots / related media on the wiki, as Github can host everything. Does that seem reasonable? I am open to counter arguments.

Reply:
Your argument definitely makes sense. My only counterpoint is that the readme is the first things people see, as it is shown on the main Github page. If we don't capture interest here, they won't click through to the wiki.

Also, the static web page would require screenshots, too, and it is usually also kept in the repo (see Github pages).